di_website/templates/includes/partials/project.html

```diff
@@ -3,17 +3,17 @@
 <div class="project">
   <div class="project__main">
     <div class="project__header">
-      {% if item.image %}
-        {% image item.image fill-180x112-c100 as image %}
+      {% if self.image %}
+        {% image self.image fill-180x112-c100 as image %}
         <div class="project__media">
           <img src="{{ image.url }}" alt="{{ image.alt }}">
         </div>
       {% endif %}
-      <h2 class="project__heading">{{item.heading}}</h2>
-      <h3 class="project__subheading">{{item.sub_heading}}</h3>
+      <h2 class="project__heading">{{self.title}}</h2>
+      <h3 class="project__subheading">{{self.subtitle}}</h3>
```

Comment:
Will we always have subtitles? If not, it may be worth piping this through `default_if_none:""` otherwise a null DB field prints the string `"None"`

Reply:
Done!